widget/src/api/user-session/login.ts

```diff
@@ -85,7 +86,8 @@ export class LoginAPI implements LoginAPICodeChallengeStorageAccessor {
     };
   }
 
-  createLoginURL(payload?: string): string {
+  createLoginWebsiteUrl( args: {payload: {}, targetAuthPage: string}): string {
```

Comment:
I think changing the payload type to `{[key: string]: string | number}` or `{path?: string}` would be better.
And providing an enum for the `targetAuthPage` type would be helpful also